source/release-notes/5.0.txt

```diff
@@ -761,6 +761,14 @@ The :dbcommand:`validate` command and :method:`db.collection.validate()`
 helper method also return a new :data:`~validate.repaired` boolean value
 that is ``true`` if the collection was repaired.
 
+Metadata Option Added for ``validate`` Command
```

Comment:
Since this isn't applicable to all minor versions of 5.0 (it's 5.0.4+), we should put this in the 5.1 release notes and say:

"Starting in MongoDB 5.1 (and MongoDB 5.0.4), ..."

Reply:
Updated and moved to 5.1 notes